cache/edge_lambdas/src/redirector.ts

```diff
@@ -84,6 +84,17 @@ export const getRedirect = (
     return redirect301(host, literalRedirects[uriSansSlash]);
   }
 
+  if (
+    !literalRedirects[uriSansSlash] &&
```

Comment:
Is this needed here? (It should've returned from the line 83 condition?)

Reply:
This was just a belt and braces approach in case the conditions got moved around. I've added a comment to say the literalRedirects must be checked first, so could take this out